lib/pbench/test/unit/server/conftest.py

```diff
@@ -759,16 +759,11 @@ def generate_token(
     # Current time to encode in the token payload
     current_utc = datetime.datetime.now(datetime.timezone.utc)
 
-    # Expired epoch time to use as a token expiry in case the caller
-    # wants an invalid token.
-    EXPIRY = "1667187770"
-
     if not user:
         user = User.query(username=username)
-    if valid:
-        exp = current_utc + datetime.timedelta(minutes=10)
-    else:
-        exp = EXPIRY
+    assert user
```

Comment:
I would have put the assertion in the block at line 763 (since, if we skip that block, then we _know_ it's not `None`), but I s'pose it's OK where it is.  😏

Reply:
Yeah, I put it there because after querying the User table we still want to make sure its not `None` because we use parameters of the user object later in the payload. So we rather fail here than trying to access user object parameters later.

Reply:
Oh, I think it's an excellent idea to have the assertion, I just would indent it so that it is executed _only_ after querying the User table -- there is no need to do the check if we don't call `User.query()`, because, in that case, we already know that the check won't fail.